### ISSUE TITLE
Not able to pass an activity to skill bot

### ISSUE THREAD

**Issue by gapmdq** [microsoft/BotFramework-Composer#8128]
## Describe the bug
I have a dialog that connects to a skill, I'm setting the activity as **${conversation.query}** like in the screenshot, to pass that variable to the bot , also setting **Activity processed as false**, but no luck, the skill is receiving what is in **turn.activity.text** anyway. 

## Version

Version 2.0
## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS
Chrome

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu
Windows
## Screenshots

![image](https://user-images.githubusercontent.com/12549664/122468705-6d3fba80-cf92-11eb-8f76-8eba0bceeff3.png)

**Comment by jwiley84** [comment]
Hi @gapmdq ! Sorry for the delay. Let me try a repro of this and get  back to you on what's going on!

**Comment by srinaath** [comment]
@gapmdq Instead of using the friendly editor. Can you use the code editor and pass the value through an activity template like this.
```
[Activity
type = event
name = AddTodo
value = ${{ "itemTitle": "my todo", "listType": "todo" }}
]
```
![Screen Shot 2021-06-22 at 10 34 12 AM](https://user-images.githubusercontent.com/13004779/122972828-6a074e80-d345-11eb-9f17-c5f010ffb51d.png)

**Comment by gapmdq** [comment]
Hi Srinaath
I'm not sure if I'm missing something but the skill is still receiving what the user input (turn.activity.text) and it is ignoring Activity value. 
For example I typed "ticket", trigger the Skilldialog , the skill dialog connects to the skill sending the activity as you mention, but the skill it is still recognizing the input value. 
Skill trace:
{
  "recognizerResult": {
    "text": "ticket",
    "alteredText": null,
    "intents": {
      "TicketClose": {
        "score": 0.49167684
      }
    },
    "entities": {}
  }
}

Thanks!

**Comment by stevkan** [comment]
@gapmdq, apologies for the delay in response. Are you still having an issue with passing an activity to a skill? I was able to successfully do so using v2.0.0 of Composer.

In short, I have a trigger in my root bot that responds to 'start skill' which then begins the 'StartSkill' dialog with a similar name. In the dialog, I send a response indicating the skill is about to start. I then connect to the skill passing the event activity @srinaath mentioned in a previous message, set allow interruptions to `true`, and set activity processed to `true`. 

In the skill, for simplicity, I created a basic event activity trigger that only sends a response showing the activity passed in from the root bot. The conversation looks like this:

![image](https://user-images.githubusercontent.com/18403946/128110594-cf14a444-95fa-4489-a3ad-577e5929deb4.png)
